docs/developer-docs/integrations/sns/get-sns/local-testing.md

```diff
@@ -1,74 +1,150 @@
 <!--# Testing SNS Locally after Choosing Parameters-->
 
 
-After having
-[chosen the initial SNS parameters in a .yaml file](preparation.md)
+After having [chosen the initial SNS parameters in a .yaml file](preparation.md)
 and before requesting an SNS launch in production,
 you might want to test the SNS launch locally.
 You might also want to do this with different SNS parameters to compare different behaviors.
 
 We next describe how you can test the SNS launch with your chosen parameters.
-The necessary steps are very similar to the [process of getting an SNS
-in production](get-sns-production.md).
+The necessary steps are very similar to the [process of getting an SNS in production](get-sns-production.md).
```

Comment:
```suggestion
The necessary steps are very similar to the process of getting an SNS in production.
```

Let's remove the link for now as this page is not yet ready (and people cannot yet get an SNS in prodcution)